packages/angular/src/generators/utils/names.ts

```diff
@@ -0,0 +1,11 @@
+import { normalizePath } from '@nrwl/devkit';
+
+export type NameInfo = { name: string; path: string };
+
+export function parseName(rawName: string): NameInfo {
```

Comment:
Is this function in devkit? How do we handle this for creating projects?

Reply:
This is how the Angular DevKit directive schematic does and what we have in the component generator (https://github.com/leosvelperez/nx/blob/master/packages/angular/src/generators/component/lib/normalize-options.ts#L15-L17). I just made sure to make it consistent and have the directive generator behave the same. It's not meant to be used for creating projects. It's a parsing that would handle the scenario in which someone specifies a path in the name. I think it makes sense to handle it. It doesn't break existing behavior and it's aligned with the Angular DevKit directive schematic.